release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3332,6 +3332,63 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, oc-mirror plugin v2 for tags were not created for the {product-title} release images. Some container registries depend on these tags as mandatory tags. With this release, these tags get added to all release images. (link:https://issues.redhat.com/browse/OCPBUGS-37757[*OCPBUGS-37757*])
+
+* Previously, extracting the IP address from the Cluster API Machine object only returned a single address. On {vmw-first}, the returned address would always be an IPv6 address and this caused issues with the `must-gather` implementation if the address was non-routable. With this release, the Cluster API Machine object returns all IP addresses, including IPv4, so that the `must-gather` issue no longer occurs on {vmw-full}. (link:https://issues.redhat.com/browse/OCPBUGS-37607[*OCPBUGS-37607*])
+
+* Previously, the installation program incorrectly required {aws-first} permissions for creating Identity and Access Management (IAM) roles for an {product-title} cluster that already had these roles. With this release, the installation program only requests permissions for roles not yet created. (link:https://issues.redhat.com/browse/OCPBUGS-37494[*OCPBUGS-37494*]) 
+
+* Previously, when you attempted to install a cluster on {rh-openstack-first} and you used special characters, such as the hash sign (`#`) in a cluster name, the Neutron API failed to tag a security group with the name of the cluster. This caused installation of the cluster to fail. With this release, the installation program uses an alternative endpoint to tag security groups and this endpoint supports the use of special characters in tag names. (link:https://issues.redhat.com/browse/OCPBUGS-37492[*OCPBUGS-37492*])  
+
+* Previously, the Dell iDRAC baseboard management controller (BMC) with the Redfish protocol caused clusters to fail on the Dell iDRAC servers. With this release, an update to the `idrac-redfish` management interface to unset the `ipxe` parameter fixed this issue. (link:https://issues.redhat.com/browse/OCPBUGS-37262[*OCPBUGS-37262*])
+
+* Previously, the `assisted-installer` did not reload new data from the `assisted-service` when the `assisted-installer` checked control plane nodes for readiness and a conflict existed with a write operation from the `assisted-installer-controller`. This conflict caused the `assisted-installer` not detect a node that was marked by the `assisted-installer` as `Ready` because the `assisted-installer` relied on older information. With this release, the `assisted-installer` can receive the newest information from the `assisted-service`, so that it the `assisted-installer` can accurately detect the status of each node. (link:https://issues.redhat.com/browse/OCPBUGS-37167[*OCPBUGS-37167*])
```

Comment:
```suggestion
* Previously, the `assisted-installer` did not reload new data from the `assisted-service` when the `assisted-installer` checked control plane nodes for readiness and a conflict existed with a write operation from the `assisted-installer-controller`. This conflict prevented the `assisted-installer` from detecting a node that was marked by the `assisted-installer` as `Ready` because the `assisted-installer` relied on older information. With this release, the `assisted-installer` can receive the newest information from the `assisted-service`, so that the `assisted-installer` can accurately detect the status of each node. (link:https://issues.redhat.com/browse/OCPBUGS-37167[*OCPBUGS-37167*])
```

Reply:
:heavy_check_mark: